exceptions/BadRequestError.js

```diff
@@ -0,0 +1,12 @@
+class BadRequestError extends Error
+{
+    constructor (message)
+    {
+        super(message);
+        this.name = 'BadRequestError';
+        this.statusCode = '400';
+        this.message = "Bad Request";
```

Comment:
this will overwrite the `super(message)`. We might want to do:

```
this.message = message || 'Bad Message';
```

Same for another error,